config.dist.py

```diff
@@ -11,7 +11,7 @@
 GEODB_PATH = '%s/.tmp/geodb.mmdb' % BASEDIR
 
 # URL to NDN hub information
-HUBS_URL = 'http://ndnmap.arl.wustl.edu/json/geocode/'
+HUBS_URL = 'http://ndndemo.arl.wustl.edu/testbedNodes.json'
```

Comment:
is this the new canonical URL for this file? it should be made available over https

Reply:
John decided to host this file on a different web server. BTW, this file comes with a new field called `ndn-up` for each hub to show whether the corresponded hub is alive.

Apparently, it cannot be served over HTTPS right now. I can ask John to take a look at it.

Reply:
Yes please. I'm not a fan of the file name either... that uppercase N... better to stick to all lowercase letters, but I'm nitpicking now.

Reply:
I can ask John to see whether he can update the file name, too... Something like `testbed-nodes.json`

Reply:
John have updated the file name...

Reply:
Yeah. I asked him to make is accessible over HTTPS, too. He said he will probably do it by Monday.

I am still confused about the error... I have no clue how to resolve it. It does not even look like to be related to the change I submitted. Any thoughts?

Reply:
Thanks! Still not reachable over https though